packages/fiori-elements-writer/test/__snapshots__/fpm.test.ts.snap

```diff
@@ -3279,7 +3279,8 @@ EntityType=\\"com.sap.gateway.srvd.dmo.sd_travel_mduu.v0001.TravelAgencyType\\">
         \\"sap.m\\": {},
         \\"sap.ui.core\\": {},
         \\"sap.ushell\\": {},
-        \\"sap.fe.templates\\": {}
+        \\"sap.fe.templates\\": {},
+        \\"sap.fe.core\\": {}
```

Comment:
FYI This diff will be impacted by https://github.com/SAP/open-ux-tools/issues/1942 when changes re-applied.